core/cmd/app.go

```diff
@@ -969,6 +969,27 @@ func NewApp(client *Client) *cli.App {
 						},
 					},
 				},
+				{
+					Name:  "solana",
+					Usage: "Commands for handling Solana transactions",
+					Subcommands: []cli.Command{
+						{
+							Name:   "create",
+							Usage:  "Send <amount> sol from node Solana account <fromAddress> to destination <toAddress>.",
+							Action: client.SolanaSendSol,
+							Flags: []cli.Flag{
+								cli.BoolFlag{
+									Name:  "force",
+									Usage: "allows to send a higher amount than the account's balance",
+								},
+								cli.StringFlag{
+									Name:  "id",
+									Usage: "chain ID",
```

Comment:
This should explain the shorthand choices `mainnet`/`testnet`/`devnet`
- [ ] improve chain id docs

Reply:
documented options for the CLI: https://github.com/smartcontractkit/chainlink/pull/6231/commits/1dc0a5a91857d999b11eca3ff7cb0f51a68fe3f6